infinoted/infinoted-options.c

```diff
@@ -1003,6 +1013,7 @@ infinoted_options_free(InfinotedOptions* options)
   g_free(options->certificate_file);
   g_free(options->certificate_chain_file);
   g_free(options->root_directory);
+  inf_ip_address_free(options->listen_address);
```

Comment:
Understood, but it's more important to stay consistent within this PR. If I'm not mistaken some glib free functions such as `g_object_unref` don't handle NULL arguments either.